docs/developer-docs/smart-contracts/advanced-features/randomness.mdx

```diff
@@ -61,13 +61,16 @@ let (randomBytes,): (Vec<u8>,) = ic_cdk::api::call(Principal.management_canister
 <AdornedTab value={"typescript"} label="TypeScript" endAdornment={<BetaChip />}>
 
 ```typescript
-import { IDL, query, update } from 'azle';
+import { IDL, update } from 'azle';
 
 export default class {
-    @update([], IDL.Float64)
-    await call('aaaaa-aa', 'raw_rand', {
-    returnIdlType: IDL.Vec(IDL.Nat8)
-});
+    @update([], IDL.Nat64)
```

Comment:
Similar to my comment above, this example has material issues that creating an Azle project and pasting this in, and looking at it in VS Code for example, would help with:

```typescript
import { call, IDL, update } from 'azle';

export default class {
    @update([], IDL.Nat64)
    async createRand(): Promise<bigint> {
        const randomInt = await call('aaaaa-aa', 'raw_rand', {
            returnIdlType: IDL.Vec(IDL.Nat8)
        });

        return randomInt;
    }
}
```